manifests/r/rcmaehl/MSEdgeRedirect/0.7.5.3/rcmaehl.MSEdgeRedirect.installer.yaml

```diff
@@ -1,20 +1,24 @@
-# Created with WinGet Releaser v2 using Komac v1.11.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
-
 PackageIdentifier: rcmaehl.MSEdgeRedirect
 PackageVersion: 0.7.5.3
 Platform:
 - Windows.Desktop
-InstallerType: exe
-Scope: machine
 InstallerSwitches:
   Silent: /wingetinstall
   SilentWithProgress: /wingetinstall
 UpgradeBehavior: install
-ReleaseDate: 2023-12-31
+ReleaseDate: 2023-12-30
 Installers:
+- Architecture: x86
+  InstallerType: zip
+  InstallerUrl: https://github.com/rcmaehl/MSEdgeRedirect/releases/download/0.7.5.3/MSEdgeRedirect.zip
+  InstallerSha256: A2A6FA53EE184BFD1A0846A9E536D80A6EE6B4B764C61937AA553012E257C603
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: MSEdgeRedirect_x86.exe
+    PortableCommandAlias: MSEdgeRedirect
 - Architecture: x64
+  InstallerType: portable

```

Comment:
This is not a portable exe
![image](https://github.com/user-attachments/assets/c612dc72-a06f-4542-8f89-c689fa799553)